app/src/components/Parkdetails.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+
+import axios from "axios";
+import { useParams } from "react-router-dom";
+
+function Popup({ requestUrl }) {
```

Comment:
nit: This function, which is the default export, is called `Popup`, but the JS module (file) is called `Parkdetails.js`. Looks like one of them should be updated?

Also, a better capitalization for the name would be `ParkDetails`, with a capital D.